### ISSUE TITLE
Cannot reproduce the YoloV3 map from the paper

### ISSUE THREAD

**Issue by rachillesf** [AlexeyAB/darknet#4214]
Hi,

I'm training the yoloV3 with the coco dataset following the instructions provided here: https://pjreddie.com/darknet/yolo/ and I couldn't get the same map provided on the paper.

For my tests I've used a machine with dual RTX2080TI, the original yolov3.config with the initial weights darknet53.conv.74, and the dataset was downloaded with the script get_coco_dataset.sh from the darknet repository.
The best accuracy I could get was 50.3%map with an input size of 600x600. This is lower than the 57.9% reported on the paper.

This is a partial plot of when I resumed the training
![imgpsh_fullsize_anim](https://user-images.githubusercontent.com/17629688/68091881-19876880-fe64-11e9-9412-b053aacec94c.png)

Has anyone more detailed instructions to train the model that can lead to the same reported results? here are some questions I have:
   - What is the hardware used for training? 
   - Was it used more data augmentation? 
   - Should I train it for how much iterations? 
   - Was applied a warm-up step?

Thanks for the help.

**Comment by deep-practice** [comment]
Me too,much worse than original weights.Any trick? @AlexeyAB

**Comment by AlexeyAB** [comment]
@deep-practice What mAP did you get on Validation and Test(evaluation server) datasets?

You should train by using this repository https://github.com/AlexeyAB/darknet with pre-trained weights file, with 1 GPU and with these settings in cfg-file:
```
[net]
batch=64
subdivisions=8
```

----

Try to reproduce `csresnext50-panet-spp-original-optimal.cfg` without any changes in cfg-file

**Train `csresnext50-panet-spp-original-optimal.cfg` on MS COCO**

1. Download ImageNet model:
* cfg https://raw.githubusercontent.com/WongKinYiu/CrossStagePartialNetworks/master/cfg/csresnext50.cfg
* weights https://drive.google.com/open?id=1IxKu5lAYCo4UpzAl5pOBIDAvC843vjn1

2. Then do 
    `./darknet partial cfg/csresnext50.cfg csresnext50.weights csresnext50.conv.75 75`
    to get pre-trained weights file `csresnext50.conv.75`

3. Get MS COCO dataset with labels in Yolo format: https://github.com/AlexeyAB/darknet/blob/master/scripts/get_coco_dataset.sh

4. set correct paths to the `train=train.txt` in `cfg/coco.data` file

5. And train this cfg-file, **without any changes in cfg-file**: https://raw.githubusercontent.com/AlexeyAB/darknet/master/cfg/csresnext50-panet-spp-original-optimal.cfg

by using command:
`./darknet detector train cfg/coco.data cfg/csresnext50-panet-spp-original-optimal.cfg csresnext50.conv.75`

----

**Then check AP on test-dev MSCOCO evaluation server**

1. download test images: http://images.cocodataset.org/zips/test2017.zip and un-pack

2. create file `test_dev.txt` that contains paths to all images from test-dataset

3. set correct paths to the `valid=test_dev.txt` in `cfg/coco.data` file


4. get detections: `./darknet detector valid cfg/coco.data cfg/csresnext50-panet-spp-original-optimal.cfg backup/csresnext50-panet-spp-original-optimal_final.weights` and to generate a result file *.json file suiting for coco eval

5. rename and zip the result file according to COCO eval

6. go to https://worksheets.codalab.org select Paricipate --> test-dev , submit the zipped file to

7. refresh and view the eval results

8. Show results: AP, AP50, AP75

**Comment by deep-practice** [comment]
Why only use 1 gpu?What's wrong with multi gpus? @AlexeyAB

**Comment by AlexeyAB** [comment]
@deep-practice For multi-GPU you should follow: https://github.com/AlexeyAB/darknet#how-to-train-with-multi-gpu
increase max_batches= and burin_in, and decrease learning_rate=.

**Comment by deep-practice** [comment]
Ok,I know.Thanks.I'll try again.

**Comment by rachillesf** [comment]
Thanks for the answer @AlexeyAB.

The config file with the original parameters is not fitting on my GTX1080TI so I had to increase the subdivisions to 24.

What GPU are you using?
